package-res/ccc/core/base/chart/chart.data.js

```diff
@@ -96,6 +116,47 @@ pvc.BaseChart
 
         if(def.debug >= 3) this.log(this.data.getInfo());
     },
+ 
+
+    // CDF603
```

Comment:
This function seems trivial. Do we need it? Or am I not getting the "name" and purpose? Is it virtual?